src/app/zap-templates/zcl/custom-types.xml

```diff
@@ -21,4 +21,20 @@ limitations under the License.
     <type id="0xF4" name="command_id" size="1" description="Command Id" discrete="true" />
     <type id="0xF5" name="node_id" size="8" description="Node Id" discrete="true" />
   </atomic>
+  <enum name="GroupKeySecurityPolicy" type="ENUM8">
+    <item name="Standard" value="0x00"/>
+    <item name="LowLatency" value="0x01"/>
+  </enum>  
+  <struct name="GroupState">
+    <item name="VendorId" type="INT16U"/>
+    <item name="VendorGroupId" type="INT16U"/>
+    <item name="GroupKeySetIndex" type="INT16U"/>
+  </struct>
+  <struct name="GroupKeys">
+    <item name="VendorId" type="INT16U"/>
+    <item name="GroupKeyIndex" type="INT16U"/>
+    <item name="GroupKeyRoot" type="INT16U"/>
```

Comment:
GroupKeyRoot seems to be uint8[16] in the spec. This seems to me like a 16-byte value rather than the 16-bit one.
Could you double-check?

Reply:
Yes, we don't support list type in struct, now this issue has been addressed in https://github.com/project-chip/zap/pull/114. I have updated this PR to switch to use array for GroupKeyRoot. 

Please note the length argument in the array does not represent the size of the list, but the length in bytes of the related attribute,  I use 254 as in the Descriptor cluster PR since this is the maximum value authorized.